client/src/js/components/ArtifactGrid.jsx

```diff
@@ -57,7 +57,8 @@ const ArtifactGrid = () => {
   return (
     <Container className={classes.cardGrid} maxWidth='lg'>
       <UtilityBar />
-      <Grid container spacing={6}>
+      {/* <Grid container spacing={6}> */}
```

Comment:
This should not be changed as it is reverting a bug fix from the last PR.

Reply:
From the project root either 
1. Use the command `git checkout HEAD -- ./client/src/js/components/ArtifactGrid.jsx`
2. Manually make the change yourself